pbj-integration-tests/build.gradle.kts

```
@@ -71,9 +72,34 @@ testing {
                 maxHeapSize = "4096m"
             }
         }
+
+        tasks.register<Test>("integTest") {
+            testClassesDirs = sources.output.classesDirs
+            classpath = sources.runtimeClasspath
+
+            shouldRunAfter(tasks.test)
+
+            useJUnitPlatform { excludeTags("FUZZ_TEST") }
+            enableAssertions = true
+        }
+
+        tasks.register<Test>("fuzzTest") {
+            testClassesDirs = sources.output.classesDirs
+            classpath = sources.runtimeClasspath
+
+            shouldRunAfter(tasks.test)
+
+            useJUnitPlatform { includeTags("FUZZ_TEST") }
+            enableAssertions = false
+        }
     }
 }
 
+tasks.test.configure() {
```

```suggestion
tasks.test {
```

Does the same, but is more concise.

---

Will change.

---

Addressed in latest commit.